addons/web/__manifest__.py

```diff
@@ -233,29 +236,34 @@
         'web.report_assets_common': [
             ('include', 'web._assets_helpers'),
 
-            'web/static/src/legacy/scss/bootstrap_overridden_report.scss',
+            'web/static/src/report/styles/bootstrap_overridden_report.scss',
```

Comment:
why not `'web/static/src/report/bootstrap_overridden_report.scss'`? I think there is no real need to scope the style files in report/

Reply:
it's because there are like 10 scss files, that felt crowded